### PR TITLE
Reduce spacing corrections

### PR DESCRIPTION
**Pull request by TalZa1810**
### 🔦 Summary
Small corrections- reduce spacing:
- `<Table/>`
- `<TableListItem/>`
- `<SelectableAccordion/>`

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [ ] Story
  - [ ] API description
  - [ ] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [ ] Component
  - [x] Visual test

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6668

**Review by justinara — CHANGES_REQUESTED** [review]
There are a few tiny issues:
1. SelectableAccordion - padding on the left is fixed, but list item still has 30px padding on the right: https://prnt.sc/xyw9q3
2. TableListItem - side paddings should be 24px from both left and right sides (when no drag handle) and 18px on the left, and 24px on the right when with enabled drag handle. Currently it has 30px on the left at all times

File: packages/wix-style-react/src/SelectableAccordion/Item/Item.st.css [hunk 1]
```diff
@@ -36,37 +36,46 @@
...
+  padding-inline-start: 0;
+  padding-inline-end: 30px;
 }
 
-.item:verticalPadding(medium) .header {
-  padding-top: value(mediumVerticalPadding);
-  padding-bottom: value(mediumVerticalPadding);
+.item:reducedSpacingAndImprovedLayout .header {
+  padding-inline-start: 0;
+  padding-inline-end: 24px;
```
Comment (TalZa1810) [inline packages/wix-style-react/src/SelectableAccordion/Item/Item.st.css:59]:
```suggestion
  padding-inline-end: 18px;
```

File: packages/wix-style-react/src/SelectableAccordion/Item/Item.st.css [hunk 2]
```diff
@@ -36,37 +36,46 @@
...
+  padding-inline-start: 0;
+  padding-inline-end: 30px;
 }
 
-.item:verticalPadding(medium) .header {
-  padding-top: value(mediumVerticalPadding);
-  padding-bottom: value(mediumVerticalPadding);
+.item:reducedSpacingAndImprovedLayout .header {
+  padding-inline-start: 0;
+  padding-inline-end: 18px;
```
Comment (TalZa1810) [inline packages/wix-style-react/src/SelectableAccordion/Item/Item.st.css:59]:
```suggestion
  padding-inline-end: 24px;
```

File: packages/wix-style-react/src/TableListItem/TableListItem.st.css [hunk 3]
```diff
@@ -23,6 +23,15 @@
   padding: 0 value(horizontalPadding);
 }
 
+.root:reducedSpacingAndImprovedLayout {
+  padding-inline-start: 24px;
```
Comment (TalZa1810) [inline packages/wix-style-react/src/TableListItem/TableListItem.st.css:27]:
```suggestion
  padding-inline-start: 18px;
```

**Review by justinara — APPROVED** [review]

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  Project Name: **wix-ui-tpa**

  You can find it under one of the following URLs:
  * https://bo.wix.com/pages/wix-ui-tpa.pr_6668/

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  Project Name: **wix-style-react**

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6668

**Pull request merged**